test/server.test.js

```diff
@@ -32,6 +33,9 @@ injectGlobal`
     display: flex;
     justify-content: center;
   }
+  .should-be-included {
+    content: 'css-foo matches regex and this rule is swallowed (current bug)'
+  }
 `
```

Comment:
I'm sure it is, right? Change the `css-foo` to be `cs-foo` and it will now appear in the output